dzz3/dzz3/Program.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace dzz3
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите одну сторону прямоугольника");
+            var m = int.Parse(Console.ReadLine());
+            Console.WriteLine("Введите вторую сторону прямоугольника");
+            var n = int.Parse(Console.ReadLine());
+            Console.WriteLine("Введите сторону квадрата");
+            var k = int.Parse(Console.ReadLine());
+
+            var vmest_m = m / k;
+            var vmest_n = n / k;
+            var result = vmest_m * vmest_n;
```

Comment:
```suggestion
            var numberOfSquaresHorizontally = m / k;
            var numberOfSquaresVertically = n / k;
            var result = numberOfSquaresHorizontally  * numberOfSquaresVertically;
```
А ещё лучше вовсе не вводить лишних (используемых только один раз) переменных
```suggestion
            var result = (m / k) * (n / k);
```